infrastructure/cron-jobs/retrieve-pnet-logs.cron.yml

```diff
@@ -49,31 +79,50 @@ objects:
                     }
 
 
+                    function create_db_indices() {
+                      # Re-create indices on PharmanetTransactionLog table for better performance
+                      # for queries/reports
+
+                      get_now_timestamp NOW_TIMESTAMP; echo -e "-------- Creating indices at ${NOW_TIMESTAMP} --------"
+                      exec_sql_no_resultset 'CREATE INDEX "IX_PharmanetTransactionLog_PharmacyId" ON public."PharmanetTransactionLog" USING btree ("PharmacyId");'
+                      get_now_timestamp NOW_TIMESTAMP; echo -e "Created IX_PharmanetTransactionLog_PharmacyId at ${NOW_TIMESTAMP}"
+                      exec_sql_no_resultset 'CREATE INDEX "IX_PharmanetTransactionLog_TxDateTime" ON public."PharmanetTransactionLog" USING btree ("TxDateTime");'
+                      get_now_timestamp NOW_TIMESTAMP; echo -e "Created IX_PharmanetTransactionLog_TxDateTime at ${NOW_TIMESTAMP}"
+                      exec_sql_no_resultset 'CREATE INDEX "IX_PharmanetTransactionLog_UserId" ON public."PharmanetTransactionLog" USING btree ("UserId");'
+                      get_now_timestamp NOW_TIMESTAMP; echo -e "Created IX_PharmanetTransactionLog_UserId at ${NOW_TIMESTAMP}"
+                    }
+
+
                     function main() {
                       echo -e "-------- STARTING CRON at $(date +"%B %d, %Y %T") UTC --------\n"
 
-                      echo -e "Connecting to database host:  _${PGHOST}_\n"
-                      echo -e "API client name:  _${PRIME_ODR_API_CLIENT_NAME}_\n"
-                      echo -e "Fetch size:  _${PRIME_ODR_API_FETCH_SIZE}_\n"
+                      echo -e "Connecting to database host:  _${PGHOST}_"
+                      echo -e "API client name:  _${PRIME_ODR_API_CLIENT_NAME}_"
+                      echo -e "Fetch size:  _${PRIME_ODR_API_FETCH_SIZE}_"
+                      echo
 
+                      drop_db_indices
                       HAS_MORE='Y'
                       while [ "${HAS_MORE}" = 'Y' ]
                       do
                         get_last_tx_id LAST_TX_ID
                         echo -e "Last transaction id:  _${LAST_TX_ID}_\n"
 
                         UUID=$(cat /proc/sys/kernel/random/uuid)
-                        echo -e "Generated request id:  _${UUID}_\n"
+                        echo -e "Generated request id:  _${UUID}_"
 
+                        echo -e "-------- Calling PRIME-ODR API then Postgres COPY --------\n"
                         # CA certs need to be in place:  https://stackoverflow.com/questions/3160909/how-do-i-deal-with-certificates-using-curl-while-trying-to-access-an-https-url
                         # ls -l /etc/ssl/certs
                         curl --cert /opt/certs/prime-odr-api-cert.crt:${PRIME_ODR_API_SSL_CERT_PASSWORD} --key /opt/certs/prime-odr-api-cert.key --header "Authorization: Basic ${PRIME_ODR_API_ENCODED_CREDENTIALS}" \
```

Comment:
I'd suggest maybe silencing curl output, so that there is no progress.
`curl -sS ...`

to avoid this:
```shell
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0100   123    0   123    0     0     21      0 --:--:--  0:00:05 --:--:--    28100   123    0   123    0     0     21      0 --:--:--  0:00:05 --:--:--    37